community/basic_examples/entanglement/entanglement.ipynb

```diff
@@ -0,0 +1,110 @@
+    "    my_bell_state(registers)\n",
+    "\n",
+    "model = create_model(main)\n",
+    "write_qmod(model, \"bell_state_of_2_qubits\")\n",
+    "\n",
+    "\n",
+    "qprog = synthesize(model)\n",
+    "\n",
+    "circuit = QuantumProgram.from_qprog(qprog)\n",
+    "\n",
+    "circuit.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Mathematical Background\n",
+    "\n",
+    "Alice has a qubit which is initially set to $|0\\rangle$<br>\n",
+    "Bob has a qubit which is initally set to $|0\\rangle$\n",
+    "\n",
+    "Alice applies 2x2 Hadamard Matrix (H) to create a superposition of her qubit's state. It is defined as:<br>\n",
+    "\n",
+    "$$H \\cdot \\begin{pmatrix} 1 \\\\ 0 \\end{pmatrix} = \\begin{pmatrix} \\frac{1}{\\sqrt{2}} & \\frac{1}{\\sqrt{2}} \\\\ \\frac{1}{\\sqrt{2}} & -\\frac{1}{\\sqrt{2}} \\end{pmatrix} \\cdot \\begin{pmatrix} 1 \\\\ 0 \\end{pmatrix} = \\begin{pmatrix} \\frac{1}{\\sqrt{2}} \\\\ \\frac{1}{\\sqrt{2}} \\end{pmatrix} = \\frac{1}{\\sqrt{2}} \\left| 0 \\right\\rangle + \\frac{1}{\\sqrt{2}} \\left| 1 \\right\\rangle$$\n",
+    "\n",
+    "Alice and Bob combine their qubits and generate the composite quantum state as:\n",
+    "$$\\begin{pmatrix} \\frac{1}{\\sqrt{2}} \\\\ \\frac{1}{\\sqrt{2}} \\end{pmatrix} \\otimes \\begin{pmatrix} 1 \\\\ 0 \\end{pmatrix} = \\begin{pmatrix} \\frac{1}{\\sqrt{2}} \\\\ 0 \\\\ \\frac{1}{\\sqrt{2}} \\\\ 0 \\end{pmatrix}$$\n",
+    "\n",
+    "Alice now applies the Controlled NOT gate operation, with her qubit being the control qubit and Bob's qubit as the target qubit. The Controlled NOT qubit when applied only affects the target qubit by inverting its state if the control qubit is $|1\\rangle$. The 4x4 matrix for Controlled NOT(CNOT) transformation matrix is defined as: <br>\n",
+    "$$\\text{CNOT} = \\begin{pmatrix}\n",
+    "1 & 0 & 0 & 0 \\\\\n",
+    "0 & 1 & 0 & 0 \\\\\n",
+    "0 & 0 & 0 & 1 \\\\\n",
+    "0 & 0 & 1 & 0\n",
+    "\\end{pmatrix}$$\n",
+    "The new quantum state is:\n",
+    "$$\\begin{pmatrix} 1 & 0 & 0 & 0 \\\\ 0 & 1 & 0 & 0 \\\\ 0 & 0 & 0 & 1 \\\\ 0 & 0 & 1 & 0 \\end{pmatrix} \\begin{pmatrix} \\frac{1}{\\sqrt{2}} \\\\ 0 \\\\ \\frac{1}{\\sqrt{2}}\\\\ 0 \\end{pmatrix} = \\begin{pmatrix} \\frac{1}{\\sqrt{2}} \\\\ 0 \\\\ 0 \\\\ \\frac{1}{\\sqrt{2}} \\end{pmatrix} = \\frac{1}{\\sqrt{2}} \\left| 00 \\right\\rangle + \\frac{1}{\\sqrt{2}} \\left| 11 \\right\\rangle$$\n",
+    "\n",
+    "In this situation the qubit's of Alice and Bob are correlated to each other.<br>\n",
+    "If we measure the both the qubits we will either get the state $|00\\rangle$ or the state $|11\\rangle$ with equal probability.\n",
+    "\n",
+    "It is described as:\n",
+    "- When the Alice oberseve her state as $|0\\rangle$ then the state of Bob's qubit collapses to the state $|0\\rangle$\n",
+    "- When the Alice oberseve her state as $|1\\rangle$ then the state of Bob's qubit collapses to the state $|1\\rangle$ \n",
```

Comment:
When the Alice oberseve -> When Alice observes

Reply:
Hey @jyotiraj-code - great work. Two small typos. If you could commit the changes I will approve and merge. Thank you!